### DISCUSSION TITLE
LCP Metric... question?

### DISCUSSION THREAD

**Discussion by meiyasan** [GoogleChrome/lighthouse#14632] [Q&A] [answered]
Hello,

I have been trying to adjust the LCP metric for quite some time now.
On one of my website, the LCP metric is getting crazy, because I am lazyloading an optional background image of size 100vw/100vh. This one is flagged as LCP, however, this is totally optional..

I have been reading quite a lot of articles about lazyloading and LCP bias... but I feel something is wrong with this metric in this context. LCP makes sense when image is loaded as blocking ressource, but in this case this image is optional.
So is there a way to ignore LCP flag ?

Any suggestions are welcome. Many thanks for your help!

**Comment by adamraine** [comment, marked as answer]
There is not a way to manually ignore a specific element for LCP.

However, [images that occupy the entire viewport aren't considered for LCP](https://web.dev/lcp/#:~:text=Images%20that%20occupy%20the%20entire%20viewport%20are%20not%20considered%20LCP%20candidates.).

**Reply by meiyasan** [reply]
I see.. that must be reason why the background is not considered but the logo is.
Thank you so much :-)
